src/styles/variables.css

```
@@ -1,80 +1,29 @@
-/* Define Custom Media queries */
-@custom-media --mq-xxs-down (max-width: 500px);
-@custom-media --mq-xs-down (max-width: 767px);
-@custom-media --mq-sm-down (max-width: 1023px);
-@custom-media --mq-md-down (max-width: 1199px);
-@custom-media --mq-lg-down (max-width: 1439px);
-@custom-media --mq-xl-down (max-width: 1919px);
+@import 'topdanmark-tokens.css';
 
-@custom-media --mq-sm-only (min-width: 768px) and (max-width: 1023px);
+/* Define Custom Media queries */
+@custom-media --mq-screen-xs-max (max-width: 479px);
+@custom-media --mq-screen-sm-max (max-width: 767px);
+@custom-media --mq-screen-md-max (max-width: 991px);
 
-@custom-media --mq-xs (min-width: 500px);
-@custom-media --mq-sm (min-width: 768px);
-@custom-media --mq-md (min-width: 1024px);
-@custom-media --mq-lg (min-width: 1200px);
-@custom-media --mq-xl (min-width: 1440px);
-@custom-media --mq-xxl (min-width: 1920px);
+@custom-media --mq-screen-sm (min-width: 480px);
+@custom-media --mq-screen-md (min-width: 768px);
+@custom-media --mq-screen-lg (min-width: 992px);
 
-/* Define global CSS variables inside :root */
+/* ============================================ */
+/* Topdanmark specific variables */
+/* ============================================ */
 :root {
-  --content-max-width: calc((100vw / 14) * 12);
-
-  @media (--mq-xxl) {
-    --content-max-width: calc((100vw / 14) * 10);
-  }
-
-  --grid-column: calc(100vw / 14);
-  --grid-row: calc(100vh / 14);
-  --grid-gutter: 1rem;
-
-  --color-indigo: #1e2db4;
-  --color-light: #f2f2f2;
-  --color-black: #000000;
-  --color-black-50: #00000080;
-  --color-black-10: #0000001a;
-  --color-white: #ffffff;
-
-  --font-family-default: 'Interstate', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
+  /* todo: Missing font token for 18px - do we need it? used lots in mobile views */
+  --font-size-m: 18px;
 
+  /* todo: Refactor to use token - hard because of bootstrap and its calculation */
   --font-size-base: 16px;
-  --font-size-sm: 0.875rem;
-  --font-size-body: 1rem;
-  --font-size-md: 1.125rem;
-  --font-size-lg: 1.25rem;
-  --font-size-xl: 1.625rem;
-  --font-size-xxl: 2.375rem;
-
-  @media (--mq-sm) {
-    --font-size-md: 1.25rem;
-    --font-size-lg: 1.625rem;
-    --font-size-xl: 2.375rem;
-    --font-size-xxl: 4rem;
-  }
-
-  --font-weight-xtralight: 100;
-  --font-weight-light: 300;
-  --font-weight-regular: 400;
-  /*
-  --font-weight-medium: 500;
-  --font-weight-demibold: 600;
-  */
-  --font-weight-bold: 700;
-
-  --lineheight-xs: 1.2em;
-  --lineheight-sm: 1.3em;
-  --lineheight-md: 1.4em;
 
-  --letterspacing-body: 0em;
+  --font-family-base: Interstate, 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Why is `Interstate, 'Helvetica Neue', Helvetica, Arial, sans-serif;` better than what previously have? If just because old flos is using this, I suggest change to something nextjs scaffold if you don't mind.

```suggestion
  --font-family-base: 'Interstate', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
```